asv_bench/benchmarks/dataarray_missing.py

```diff
@@ -1,8 +1,7 @@
 import pandas as pd
 
 import xarray as xr
-
-from . import parameterized, randn, requires_dask
+from asv_bench.benchmarks import parameterized, randn, requires_dask
```

Comment:
This makes our benchmarks fail unfortunately:
```
    STDERR -------->
Error:    Traceback (most recent call last):
     File "/home/runner/micromamba-root/envs/xarray-tests/lib/python3.10/site-packages/asv/benchmark.py", line 1435, in <module>
       main()
     File "/home/runner/micromamba-root/envs/xarray-tests/lib/python3.10/site-packages/asv/benchmark.py", line 1428, in main
       commands[mode](args)
     File "/home/runner/micromamba-root/envs/xarray-tests/lib/python3.10/site-packages/asv/benchmark.py", line 1103, in main_discover
       list_benchmarks(benchmark_dir, fp)
     File "/home/runner/micromamba-root/envs/xarray-tests/lib/python3.10/site-packages/asv/benchmark.py", line 1088, in list_benchmarks
       for benchmark in disc_benchmarks(root):
     File "/home/runner/micromamba-root/envs/xarray-tests/lib/python3.10/site-packages/asv/benchmark.py", line 985, in disc_benchmarks
       for module in disc_modules(root_name, ignore_import_errors=ignore_import_errors):
     File "/home/runner/micromamba-root/envs/xarray-tests/lib/python3.10/site-packages/asv/benchmark.py", line 967, in disc_modules
       for item in disc_modules(name, ignore_import_errors=ignore_import_errors):
     File "/home/runner/micromamba-root/envs/xarray-tests/lib/python3.10/site-packages/asv/benchmark.py", line 950, in disc_modules
       module = import_module(module_name)
     File "/home/runner/work/xarray/xarray/asv_bench/.asv/env/3e1d7de4e47af51e3e41695ae1884ae2/lib/python3.8/importlib/__init__.py", line 127, in import_module
       return _bootstrap._gcd_import(name[level:], package, level)
     File "<frozen importlib._bootstrap>", line 1014, in _gcd_import
     File "<frozen importlib._bootstrap>", line 991, in _find_and_load
     File "<frozen importlib._bootstrap>", line 975, in _find_and_load_unlocked
     File "<frozen importlib._bootstrap>", line 671, in _load_unlocked
     File "<frozen importlib._bootstrap_external>", line 843, in exec_module
     File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
     File "/home/runner/work/xarray/xarray/asv_bench/benchmarks/dataarray_missing.py", line 4, in <module>
       from asv_bench.benchmarks import parameterized, randn, requires_dask
   ModuleNotFoundError: No module named 'asv_bench'
```

Reply:
Hey, thanks for using `absolufy-imports` - just for reference, in pandas we don't use it on `asv_bench`

https://github.com/pandas-dev/pandas/blob/4eb4729fb47bd160a39ec49f6af89dabfc63d3ac/.pre-commit-config.yaml#L8-L12

If you add a line to the hook with `- files: ^xarray/`, and revert the changes to `asv_bench`, then I think it should work fine